dapps/templates/boilerplate/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "embark-dapp-template-boilerplate",
+  "version": "4.0.0-beta.0",
+  "description": "",
+  "files": [
+    "app",
+    "config",
+    "contracts",
+    "dot.gitignore",
+    "dot.npmrc",
+    "embark.json",
+    "test"
+  ],
+  "scripts": {
+    "ci": "npm run qa",
+    "clean": "npm run reset",
+    "package": "npm pack",
+    "qa": "npm-run-all test package",
+    "reset": "npx embark-reset && npx rimraf embark-*.tgz package",
```

Comment:
is `rimraf` really necessary to delete the tarball? Could be one less dependency needed if we could remove it from each of the templates. I do understand `embark-reset` needs `rimraf` anyway, but removing it from the template dependencies just make it easier to read an understand.

Reply:
It gets removed (deleted from `package.json`) when the template is instantiated.  As a devDependency in the workspace, it's lightweight because it gets hoisted into the root `node_modules` and is effectively shared by many of the other monorepo packages that depend on the same version.